README.md

```diff
@@ -1,28 +1,156 @@
-        array('$email' => 'someone@example.com', '$first_name' => 'Bill', '$last_name' => 'Shakespeare'),
-        array('Item SKU' => 'ABC123', 'Payment Method' => 'Credit Card'),
-        1354913220
-    );
+* Track customers and events directly from your backend.
 
-Argument descrption
--------------
 
-The `track` method takes four arguments:
+## How to install?
 
-**event** This is the name of the event you want to track. It can be any string you like.
+    composer require klaviyo/php-sdk
+    
+## API Examples
 
-**customer_properties** This is an associative array of properties that belong to the person who did the action you're recording. You must include either a `$id` or `$email` key.
+After installing the klaviyo package you can initiate it using your public token which is for track events or identifying profiles and/or your private api key to utilize the metrics and list apis.
+```php
+use Klaviyo\Klaviyo as Klaviyo;
 
-**properties** (optional) This is an associative array of properties that are specific to this event occurrance. In the above example, we included the SKU of the item that was purchased and the payment method.
+$client = new Klaviyo( 'PRIVATE_API_KEY', 'PUBLIC_API_KEY' );
+```
+You can then easily use Klaviyo to track events or identify people.  Note, track and identify requests take your public token.
 
-**timestamp** (optional) This is the timestamp when this event occurred. You only need to include this if you're tracking events that occurred in the past. If you're tracking activity in real time, you can ignore this argument.
\ No newline at end of file
+
+### Track an event...
+```php
+use Klaviyo\Model\EventModel as KlaviyoEvent;
+    
+$event = new KlaviyoEvent(
+    array(
+        'event' => 'Filled out Profile',
+        'customer_properties' => array(
+            '$email' => 'someone@mailinator.com'    
+        ),
+        'properties' => array(
+            'Added Social Accounts' => False
+        )
+    )
+);
+    
+$client->track->trackEvent( $event );    
+```    
+### You can also add profile properties to the 'customer properties' attribute in the Event model
+```php
+use Klaviyo\Model\EventModel as KlaviyoEvent;
+        
+$event = new KlaviyoEvent(
+    array(
+        'event' => 'Filled out Profile',
+        'customer_properties' => array(
+            '$email' => 'someone@mailinator.com',
+            '$first_name' => 'Thomas',
+            '$last_name' => 'Jefferson'   
+        ),
+        'properties' => array(
+            'Added Social Accounts' => False
+        )
+    )
+);
+
+$client->track->trackEvent( $event );
+```
+
+### or just add a property to someone
+```php
+use Klaviyo\Model\ProfileModel as KlaviyoProfile;
+    
+$profile = new KlaviyoProfile(
+    array(
+        '$email' => 'thomas.jefferson@mailinator.com',
+        '$first_name' => 'Thomas',
+        '$last_name' => 'Jefferson',
+        'Plan' => 'Premium'
+    )
+);
+    
+$client->track->identifyProfile( $profile );
```

Comment:
i think we should change this config so it's something like:
```
$client->public->identify()
and
$client->public->track()
```

Reply:
I like that, great call!

Reply:
👍

Reply:
I don't think we can name a class public, Going to name it PublicAPI